flow/src/test/scala/org/alephium/flow/mining/MinerSpec.scala

```diff
@@ -20,14 +20,15 @@ import scala.annotation.tailrec
 
 import akka.testkit.TestActorRef
 import akka.util.Timeout
+import org.scalatest.concurrent.Eventually.eventually
 import org.scalatest.concurrent.ScalaFutures
 
 import org.alephium.flow.{AlephiumFlowActorSpec, FlowFixture}
 import org.alephium.flow.handler.{BlockChainHandler, TestUtils, ViewHandler}
 import org.alephium.flow.model.MiningBlob
 import org.alephium.flow.validation.InvalidBlockVersion
-import org.alephium.protocol.model._
-import org.alephium.serde._
+import org.alephium.protocol.model.*
+import org.alephium.serde.*
```

Comment:
should we restrict to `_` for consistency?

Reply:
yes, I will fix this

Reply:
Fixed!